packages/core/src/types.ts

```diff
@@ -239,6 +241,10 @@ export const OutputMode = {
 
 export type OutputMode = (typeof OutputMode)[keyof typeof OutputMode];
 
+export type OutputDocsOptions = {
+  config: string;
```

Comment:
I thought that being able to define `TypeDocOptions` directly here instead of in a configuration file might have the advantage of reducing the number of configuration files to manage.
Is it okay to explicitly indicate that it is a file path, considering the expandability of the option in the future?

```suggestion
  configPath: string;
```

Reply:
I've renamed the option to `configPath` and also added support for passing the TypeDoc options directly inside the `docs` object.

Reply:
That's nice!